src/UnoFlipController.java

```diff
@@ -53,29 +51,50 @@ public void actionPerformed(ActionEvent e) {
                     null,
                     null);
 
+            // numPlayers used to create the amount of players the user has selected
             int numPlayers = 0;
-            if (result == JOptionPane.OK_OPTION) {
+            if (result == JOptionPane.OK_OPTION)
+            {
+                // Parse the input given by the user from the drop-down to use to create players
                 String option = (String) comboBox.getSelectedItem();
                 numPlayers = Integer.parseInt(option);
+                // Tell model the number of players in the game
                 this.model.setNumPlayers(numPlayers);
-            } else {
+            }
+            else // Exit the game if the user cancels
+            {
                 System.exit(0); // absolutely must select an option, otherwise, do not start the game
             }
 
-            for (int i = 0; i < numPlayers; i++){
+            // For the number of players prompt the user for name input
```

Comment:
Again, you're somewhat translating what the code is doing. Instead, I think it is more clear to say "Prompt the user for the name of each player". This is a bit more clear than "For the number of players prompt the user for name input"